types/autoprefixer/index.d.ts

```diff
@@ -1,12 +1,12 @@
-// Type definitions for autoprefixer 6.7
+// Type definitions for autoprefixer 9.1
 // Project: https://github.com/postcss/autoprefixer
-// Definitions by: Armando Meziat <https://github.com/odnamrataizem>
+// Definitions by:  Armando Meziat <https://github.com/odnamrataizem>, murt <https://github.com/murt>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-import { Plugin, Transformer as PostcssTransformer } from 'postcss';
+import { Plugin } from "postcss";
 
 declare namespace autoprefixer {
-    interface Options {
+    interface AutoprefixerOptions {
```

Comment:
It seems like changing the name of this interface is just going to break people without much benefit. It's already insider `namespace autoprefixer` so I don't think adding `Autoprefixer` to the name is necessary anyway.

Reply:
Good catch, updated.